.gitignore

```diff
@@ -9,6 +9,7 @@ bower_components/
 coverage/
 dist/
 kubernetes/
+tools/
```

Comment:
You're correct. gulp clean should not clean godep. The thing that struck me now is why we download kubernetes cluster in gulp and godep in npm. Should there be this difference? Maybe unify this? (possibly a work for future PR)